.github/labels.yml

```diff
@@ -0,0 +1,277 @@
+# ##### pull-request-size bot labels ##### #
+- color: "009900" # green
+  description: PR that changes 0-9 lines. Quick fix/merge.
+  name: size/XS
+- color: 77bb00 # yellow-green
+  description: PR that changes 10-29 lines. Very easy to review.
+  name: size/S
+- color: eebb00 # yellow
+  description: PR that changes 30-99 lines. Good size to review.
+  name: size/M
+- color: ee9900 # yellow-orange
+  description: PR that changes 100-499 lines. Requires some effort to review.
+  name: size/L
+- color: ee5500 # orange
+  description: PR that changes 500-999 lines. Consider splitting work into several ones that easier to review.
+  name: size/XL
+- color: ee0000 # red
+  description: PR that changes 1000+ lines. You should absolutely split your PR into several.
+  name: size/XXL
+
+# ##### complexity labels ##### #
+- color: bfe5bf # green
+  description: ""
+  name: complexity:easy
+- color: F0DA27 # yellow
+  description: ""
+  name: complexity:medium
+- color: fad8c7 # red
+  description: ""
+  name: complexity:hard
+
+# ##### TSC labels ##### #
+- color: f58220
+  description: PR managing GOVERNANCE or other TSC documentation
+  name: TSC
+- color: f58220
+  description: Items related to Technical Steering Committee voting
+  name: TSC:voting
+
+# ##### type labels ##### #
+- color: b60205
+  description: ""
+  name: breaking change
+- color: d73a4a # red (standard github)
+  description: Something isn't working
+  name: bug
+- color: d73a4a
+  description: ""
+  name: bug fix
+- color: e5b479
+  description: ""
+  name: deprecation
+- color: a2eeef # light teal (standard github)
+  description: New feature or request
+  name: enhancement
+- color: 207de5
+  description: ""
+  name: feature
+- color: bfd4f2
+  description: ""
+  name: proposal
+- color: d876e3 # pink (standard github)
+  description: Further information is requested
+  name: question
+- color: fbca04
+  description: ""
+  name: refactor
+- color: d93f0b
+  description: ""
+  name: regression
+
+# ##### flag labels ##### #
+- color: 7057ff # purple (standard github)
+  description: Good for newcomers
+  name: good first issue
+- color: 008672 # teal (standard github)
+  description: Extra attention is needed
+  name: help wanted
+- color: ffffff
+  description: ""
+  name: stale
+- color: bfdadc
+  description: ""
+  name: WIP
+
+# ##### status labels ##### #
+- color: cfd3d7 # gray (standard github)
+  description: This issue or pull request already exists
+  name: duplicate # status:duplicate
+- color: e4e669 # yellow (standard github)
+  description: This doesn't seem right
+  name: invalid # status:invalid
+- color: ffffff # white (standard github)
+  description: This will not be worked on
+  name: wontfix # status:wontfix
```

Comment:
I think these three labels should have the `status:` prefix like the rest of the status labels.